test:firefox Geckodriver don't run

Hi everyone !

After a bug in my project i try a lot of thing and so i try to fork example project [WITHOUT CHANGE ANYTHING]
And when i run `npm run test:firefox`

i get this error : 

```powershell
> nightwatch-api-cucumber@1.0.0 test:firefox D:\Projets\Projet_Test\nightwatch-api\packages\cucumber-example
> cross-env browser=firefox npm test


> nightwatch-api-cucumber@1.0.0 test D:\Projets\Projet_Test\nightwatch-api\packages\cucumber-example
> cucumber-js --require-module babel-core/register --require cucumber.conf.js --require step-definitions --format node_modules/cucumber-pretty

VError: a BeforeAll hook errored, process exiting: cucumber.conf.js:6: function timed out, ensure the promise resolves within 60000 milliseconds
    at D:\Projets\Projet_Test\nightwatch-api\packages\cucumber-example\node_modules\cucumber\lib\runtime\index.js:68:19
    at Generator.next (<anonymous>)
    at asyncGeneratorStep (D:\Projets\Projet_Test\nightwatch-api\packages\cucumber-example\node_modules\cucumber\lib\runtime\index.js:26:103)
    at _next (D:\Projets\Projet_Test\nightwatch-api\packages\cucumber-example\node_modules\cucumber\lib\runtime\index.js:28:194)
caused by: Error: function timed out, ensure the promise resolves within 60000 milliseconds
    at Timeout._time.default.setTimeout [as _onTimeout] (D:\Projets\Projet_Test\nightwatch-api\packages\cucumber-example\node_modules\cucumber\lib\user_code_runner.js:81:20)
    at listOnTimeout (timers.js:324:15)
    at processTimers (timers.js:268:5)
npm ERR! Test failed.  See above for more details.
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! nightwatch-api-cucumber@1.0.0 test:firefox: `cross-env browser=firefox npm test`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the nightwatch-api-cucumber@1.0.0 test:firefox script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     d:\Users\frup72768\AppData\Roaming\npm-cache\_logs\2019-03-11T09_42_04_439Z-debug.log
PS D:\Projets\Projet_Test\nightwatch-api\packages\cucumber-example>
```

Firefox version 65.0.2

```
 "devDependencies": {
    "babel-cli": "^6.26.0",
    "babel-core": "^6.26.3",
    "babel-preset-env": "^1.7.0",
    "chromedriver": "2.46.0",
    "cross-env": "^5.2.0",
    "cucumber": "^5.1.0",
    "cucumber-pretty": "^1.5.0",
    "geckodriver": "^1.15.1",
    "nightwatch": "^1.0.19",
    "nightwatch-api": "latest"
  }
```

--- a/packages/cucumber-example/nightwatch.conf.js
+++ b/packages/cucumber-example/nightwatch.conf.js
@@ -2,6 +2,7 @@ const chromedriver = require('chromedriver');
 const geckodriver = require('geckodriver');
 
 module.exports = {
+  silent: !process.env.NIGHTWATCH_VERBOSE,
   test_settings: {
     default: {
       webdriver: {
@@ -40,7 +41,7 @@ module.exports = {
     firefox: {
       webdriver: {
         server_path: geckodriver.path,
-        cli_args: ['--port', '4444']
+        cli_args: ['--port', '4444', '--log', 'debug']
       },
       desiredCapabilities: {
         browserName: 'firefox',

--- a/packages/cucumber-example/package.json
+++ b/packages/cucumber-example/package.json
@@ -6,7 +6,8 @@
   "scripts": {
     "test:chrome": "cross-env NIGHTWATCH_ENV=chrome npm test",
     "test:firefox": "cross-env NIGHTWATCH_ENV=firefox npm test",
-    "test": "cucumber-js --require-module babel-core/register --require cucumber.conf.js --require step-definitions --format node_modules/cucumber-pretty"
+    "test": "cucumber-js --require-module babel-core/register --require cucumber.conf.js --require step-definitions --format node_modules/cucumber-pretty",
+    "debug": "cross-env DEBUG=nightwatch-api NIGHTWATCH_ENV=firefox NIGHTWATCH_VERBOSE=true node --inspect-brk ./node_modules/cucumber/bin/cucumber-js --require-module babel-core/register --require cucumber.conf.js --require step-definitions --format node_modules/cucumber-pretty"
   },
   "repository": "git@github.com:mucsi96/nightwatch-api.git",
   "author": "Igor Mucsicska <mucsi_96@yahoo.com>",

--- a/packages/jest-example/nightwatch.conf.js
+++ b/packages/jest-example/nightwatch.conf.js
@@ -2,17 +2,18 @@ const chromedriver = require('chromedriver');
 const geckodriver = require('geckodriver');
 
 module.exports = {
+  silent: !process.env.NIGHTWATCH_VERBOSE,
   test_settings: {
     default: {
       webdriver: {
         start_process: true,
-        port: 4444,
-        cli_args: ['--port=4444']
+        port: 4444
       }
     },
     chromeHeadless: {
       webdriver: {
-        server_path: chromedriver.path
+        server_path: chromedriver.path,
+        cli_args: ['--port=4444']
       },
       desiredCapabilities: {
         browserName: 'chrome',
@@ -25,7 +26,8 @@ module.exports = {
     },
     chrome: {
       webdriver: {
-        server_path: chromedriver.path
+        server_path: chromedriver.path,
+        cli_args: ['--port=4444']
       },
       desiredCapabilities: {
         browserName: 'chrome',
@@ -38,7 +40,8 @@ module.exports = {
     },
     firefox: {
       webdriver: {
-        server_path: geckodriver.path
+        server_path: geckodriver.path,
+        cli_args: ['--port', '4444', '--log', 'debug']
       },
       desiredCapabilities: {
         browserName: 'firefox',

--- a/packages/jest-example/package.json
+++ b/packages/jest-example/package.json
@@ -4,7 +4,9 @@
   "description": "Demonstration for using Nightwatch API from Jest test runner",
   "main": "index.js",
   "scripts": {
-    "test": "jest"
+    "test": "jest",
+    "test:firefox": "cross-env NIGHTWATCH_ENV=firefox npm test",
+    "debug": "cross-env NIGHTWATCH_VERBOSE=true DEBUG=nightwatch-api node --inspect-brk ./node_modules/jest/bin/jest.js --runInBand"
   },
   "repository": "git@github.com:mucsi96/nightwatch-api.git",
   "author": "Igor Mucsicska <mucsi_96@yahoo.com>",

--- a/packages/nightwatch-api/e2e-test/nightwatch.conf.js
+++ b/packages/nightwatch-api/e2e-test/nightwatch.conf.js
@@ -12,8 +12,7 @@ module.exports = {
       launch_url: 'http://localhost:3000',
       webdriver: {
         start_process: true,
-        port: 4444,
-        cli_args: ['--port=4444']
+        port: 4444
       },
       screenshots: {
         enabled: true,
@@ -22,7 +21,8 @@ module.exports = {
     },
     chromeHeadless: {
       webdriver: {
-        server_path: chromedriver.path
+        server_path: chromedriver.path,
+        cli_args: ['--port=4444']
       },
       desiredCapabilities: {
         browserName: 'chrome',
@@ -35,7 +35,8 @@ module.exports = {
     },
     chrome: {
       webdriver: {
-        server_path: chromedriver.path
+        server_path: chromedriver.path,
+        cli_args: ['--port=4444']
       },
       desiredCapabilities: {
         browserName: 'chrome',
@@ -48,7 +49,8 @@ module.exports = {
     },
     firefox: {
       webdriver: {
-        server_path: geckodriver.path
+        server_path: geckodriver.path,
+        cli_args: ['--port', '4444', '--log', 'debug']
       },
       desiredCapabilities: {
         browserName: 'firefox',

--- a/packages/node-example/nightwatch.conf.js
+++ b/packages/node-example/nightwatch.conf.js
@@ -2,17 +2,18 @@ const chromedriver = require('chromedriver');
 const geckodriver = require('geckodriver');
 
 module.exports = {
+  silent: !process.env.NIGHTWATCH_VERBOSE,
   test_settings: {
     default: {
       webdriver: {
         start_process: true,
-        port: 4444,
-        cli_args: ['--port=4444']
+        port: 4444
       }
     },
     chromeHeadless: {
       webdriver: {
-        server_path: chromedriver.path
+        server_path: chromedriver.path,
+        cli_args: ['--port=4444']
       },
       desiredCapabilities: {
         browserName: 'chrome',
@@ -25,7 +26,8 @@ module.exports = {
     },
     chrome: {
       webdriver: {
-        server_path: chromedriver.path
+        server_path: chromedriver.path,
+        cli_args: ['--port=4444']
       },
       desiredCapabilities: {
         browserName: 'chrome',
@@ -38,7 +40,8 @@ module.exports = {
     },
     firefox: {
       webdriver: {
-        server_path: geckodriver.path
+        server_path: geckodriver.path,
+        cli_args: ['--port', '4444', '--log', 'debug']
       },
       desiredCapabilities: {
         browserName: 'firefox',

--- a/packages/node-example/package.json
+++ b/packages/node-example/package.json
@@ -5,13 +5,15 @@
   "main": "index.js",
   "scripts": {
     "test": "node test.js",
-    "debug": "node --inspect-brk test.js"
+    "test:firefox": "cross-env NIGHTWATCH_ENV=firefox npm test",
+    "debug": "cross-env NIGHTWATCH_VERBOSE=true DEBUG=nightwatch-api node --inspect-brk test.js"
   },
   "repository": "git@github.com:mucsi96/nightwatch-api.git",
   "author": "Igor Mucsicska <mucsi_96@yahoo.com>",
   "license": "MIT",
   "devDependencies": {
     "chromedriver": "2.46.0",
+    "cross-env": "^5.2.0",
     "geckodriver": "^1.16.0",
     "nightwatch": "^1.0.19",
     "nightwatch-api": "latest"

